sbws/lib/v3bwfile.py

```diff
@@ -1201,27 +1242,27 @@ def is_min_perc(self):
 
     @property
     def sum_bw(self):
-        return sum([l.bw for l in self.bw_lines])
+        return sum([l.bw for l in self.bw_lines if getattr(l, 'bw', None)])
 
     @property
     def num(self):
         return len(self.bw_lines)
 
     @property
     def mean_bw(self):
-        return mean([l.bw for l in self.bw_lines])
+        return mean([l.bw for l in self.bw_lines if getattr(l, 'bw', None)])
```

Comment:
This one needs to be hasattr rather than getattr, I think, or else the mean won't include any nodes with bw=0.